### PR TITLE
[Zero Trust] Gateway changes

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 1]
```diff
@@ -119,6 +119,47 @@ When the toggle is enabled, users have the option to turn off the [Gateway with
 1. Scroll down to **L7 Firewall**.
 1. Set the **Proxy** switch to *Enabled*.
 
+## TLS decryption
+
+This setting requires the Proxy switch to be enabled.
+
+1. Navigate to **Settings** > **Network**.
+1. Scroll down to **L7 Firewall**.
+1. Set the **Proxy** switch *Enabled*.
```
Comment (pedrosousa) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]:
```suggestion
1. Set the **Proxy** switch to *Enabled*.
```

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 2]
```diff
@@ -119,6 +119,47 @@ When the toggle is enabled, users have the option to turn off the [Gateway with
...
+:ECDHE-ECDSA-AES128-GCM-SHA256\
+:ECDHE-ECDSA-AES256-GCM-SHA384\
+:ECDHE-RSA-AES128-GCM-SHA256\
+:ECDHE-RSA-AES256-GCM-SHA384\
+:ECDHE-RSA-AES128-SHA\
+:ECDHE-RSA-AES256-SHA384\
+:AES128-GCM-SHA256\
+:AES256-GCM-SHA384\
+:AES128-SHA\
+:AES256-SHA";
```
Comment (pedrosousa) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]: Seems strange that we're closing double quotes (`"`) without opening them. Can you check?

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 3]
```diff
@@ -119,6 +119,47 @@ When the toggle is enabled, users have the option to turn off the [Gateway with
...
+
+```txt
+:CHACHA20-POLY1305-SHA256\
+:ECDHE-ECDSA-AES128-GCM-SHA256\
+:ECDHE-ECDSA-AES256-GCM-SHA384\
+:ECDHE-RSA-AES128-GCM-SHA256\
+:ECDHE-RSA-AES256-GCM-SHA384\
+:ECDHE-RSA-AES256-SHA384\
+:AES128-GCM-SHA256\
+:AES256-GCM-SHA384";
```
Comment (pedrosousa) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]: Also closing quotes here.

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 4]
```diff
@@ -119,6 +119,47 @@ When the toggle is enabled, users have the option to turn off the [Gateway with
...
 
+## TLS decryption
+
+This setting requires the Proxy switch to be enabled.
+
+1. Navigate to **Settings** > **Network**.
+1. Scroll down to **L7 Firewall**.
+1. Set the **Proxy** switch *Enabled*.
+
+TLS decryption can use both TLS version 1.2 and 1.3. The following is a list of cipher suites used when the toggle is enabled.
```
Comment (pedrosousa) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md]: Nit: You called it a "switch" and not a toggle two lines above this one.

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md [hunk 5]
```diff
@@ -119,6 +119,47 @@ When the toggle is enabled, users have the option to turn off the [Gateway with
...
+:AES256-GCM-SHA384\
+:AES128-SHA\
+:AES256-SHA";
+```
+
+### FIPS compliance
+
+When enabling TLS decryption, you have the option to only enable cipher suites and TLS versions compliant with FIPS 140-2. FIPS compliance mode requires TLS version 1.2.
+
+The following is a list of cipher suites used for TLS decryption when the **FIPS compliance** setting is enabled.
```
Comment (pedrosousa) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:152]: It's not entirely clear if **FIPS compliance** is a secondary switch/toggle.

**Review by pedrosousa — APPROVED** [review]

File: products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md (hunk 5 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/warp-settings.md:152]: I can add a screenshot of the feature! good catch.

**Comment by abracchi-tw** [comment]
changes addressed in #3474

**Pull request closed without merging**